src/test/java/com/wikia/webdriver/elements/mercury/components/discussions/desktop/Promoting.java

```diff
@@ -1,13 +1,25 @@
 package com.wikia.webdriver.elements.mercury.components.discussions.desktop;
 
+import com.google.common.base.Predicate;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject;
 
+import lombok.Getter;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.FluentWait;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import javax.annotation.Nullable;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'javax.annotation.Nullable'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)